source/conf.py

```diff
@@ -104,8 +105,27 @@
 html_theme = "sphinx_rtd_theme"
 html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
 
+with open('namespace_map.yml', 'r') as f:
+    html_context = yaml.safe_load(f)
+
+
+def rstjinja(app, docname, source):
+    """
+    Render our pages as a jinja template for fancy templating goodness.
+    """
+    # Make sure we're outputting HTML
+    if app.builder.format != 'html':
```

Comment:
I'm probably just misinterpreting something here, but I figured I should ask anyways. Since you are checking ``if app.builder.format != 'html`` here, does this mean jinja replacements are only applied when we render to HTML. If so, we should at least also support latex for building of docs in PDF format, but ideally we should not restrict the build formats we support, but leave that up to Sphinx. Since the replacements we are doing are simple, unformatted strings, I don't think this should be an issue.

Reply:
Yes that is what it means. I copied that from the tutorial I used. It makes sense to support other output formats. Hopefully this can be lifted but it may require some work arounds.

Reply:
This extension may help https://pypi.org/project/sphinx-jinja/

Reply:
yeah I looked into that but I don't think it's pip installable. I ended up following [this](https://www.ericholscher.com/blog/2016/jul/25/integrating-jinja-rst-sphinx/) tutorial, which does not require any additional dependencies